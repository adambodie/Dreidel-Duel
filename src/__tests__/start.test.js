import React from "react"
import renderer from "react-test-renderer"
import Start from "../components/start";
import {shallow} from "enzyme"

describe("Start", () =>
	it("renders correctly", () => {
		const tree = renderer.create(<Start />).toJSON();
		expect(tree).toMatchSnapshot();
	}),
	it('renders a `button`', () => {
		const wrapper = shallow(<Start />);
		expect(wrapper.find('button').length).toBe(1);
	})		
)



