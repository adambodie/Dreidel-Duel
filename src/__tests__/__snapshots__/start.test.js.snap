// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start renders correctly 1`] = `
<button
  className="action"
  onClick={[Function]}
>
  Start
</button>
`;
